crates/testing/src/view_generator.rs

```diff
@@ -577,7 +577,7 @@ impl TestViewGenerator {
         }
     }
 
-    pub async fn next_from_anscestor_view(&mut self, ancestor: TestView) {
+    pub async fn next_from_ancestor_view(&mut self, ancestor: TestView) {
```

Comment:
After grepping through our company repos I don't think this is used anywhere so shouldn't break anything.